src/Microsoft.DotNet.ProjectModel/PackOptions.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class PackOptions
```

Comment:
Do you mean this https://github.com/dotnet/core-proposals/blob/master/DotNetCore-ProjectModel.md#project-metadata? 
I just moved those mentioned here https://github.com/dotnet/core-proposals/blob/master/DotNetCore-ProjectModel.md#pack-options